src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js

```diff
@@ -1,30 +1,142 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
+import moment from "moment";
 import PropTypes from "prop-types";
-import { Header, Footer, Button } from "shared-components";
+import { Header, Footer, Button, Table, Alert } from "shared-components";
+import { MdCancel } from "react-icons/md";
 import axios from "axios";
 import { propTypes } from "../../../types/propTypes";
 
 export default function PackageReview({ page: { header, packageId } }) {
+  const [error, setError] = useState(false);
+  const [packageNoDetails, setPackageNoDetails] = useState([
```

Comment:
It reads like there is no details for the package, should it be packageDetails?

Reply:
I had that originally, but there are actually two tables side by side for the Package Details section, each with 3 key/value pairs.  One table for the Package Number and the other table for the Court File Number.  But I'm totally game to change the names to whatever makes most sense.

Reply:
```suggestion
  const [packageDetails, setPackageDetails] = useState([
```

maybe just packageDetails ?

Reply:
Done, refactored.